### PR TITLE
#161396644 Create the File Structure and Database

### PR DESCRIPTION
**Pull request by KMaina**
#### What does this PR do? 
Create the basic file structure and database tables
#### Description of Task to be completed? 
Create the file structure for the app
Create a database and populate the tables
#### How should this be manually tested?  
- Clone the repo locally 
- Ensure that `store_manager` and `test_store_manager` are created in your POSTGRESQL database
- Run the app and tables should be created
#### Any background context you want to provide? 
This is the template from which other branches will be based off 
#### What are the relevant pivotal tracker stories? 
[#161396644](https://www.pivotaltracker.com/story/show/161396644)

### CODE REVIEW

File: app/api/v2/resource/models/db.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    connection.commit()
+
+    cursor.close()
+
+    connection.close()
+
+    print('Done')
+
+if __name__ == '__main__':
+    main()
```
Comment (ghost) [inline app/api/v2/resource/models/db.py]: Add a new line here.

**Comment by ghost** [comment]
@KMaina Fix your codacy and travis builds.

**Comment by ghost** [comment]
Why is, `Any background context you want to provide?` and `What are the relevant pivotal tracker stories?` formated differently in your PR description?

File: requirements.txt [hunk 2]
```diff
@@ -1,34 +1 @@
...
-pylint==2.1.1
-pytest==3.8.2
-pytz==2018.5
-requests==2.20.0
-six==1.11.0
-typed-ast==1.1.0
-urllib3==1.24
-Werkzeug==0.14.1
-wrapt==1.10.11
+psycopg2==2.7.5
```
Comment (EdwardMudaida) [inline requirements.txt]: Why did you remove the rest of the packages?

Comment (ghost) [inline requirements.txt:24]: Why did you remove the rest of the packages?

Comment (KMaina) [reply requirements.txt:24]: Fixed
